common/server_room.cpp

```diff
@@ -36,6 +36,33 @@ Server_Room::~Server_Room()
     usersLock.unlock();
 }
 
+bool Server_Room::userMayJoin(const ServerInfo_User & userInfo)
+{
+    
+    ServerInfo_Room roomInfo;
```

Comment:
I swear I just answered this.... its used on line 624 of the protocol handler.
https://github.com/Cockatrice/Cockatrice/pull/2484/files#diff-d3b638a03e1222b79c2b7b5b8935096fR624